x-pack/plugins/ml/public/application/model_management/test_models/models/index_input.tsx

```diff
@@ -51,26 +54,43 @@ export const InferenceInputFormIndexControls: FC<Props> = ({
   const pipeline = useObservable(inferrer.getPipeline$(), inferrer.getPipeline());
   const inputComponent = useMemo(() => inferrer.getInputComponent(), [inferrer]);
 
+  // Because options are grouped, we need to filter out nested options
+  const selectedOptions: Array<EuiComboBoxOptionOption<string>> = dataViewListItems
+    .map((v) => v.options)
+    .flat()
+    .filter((v) => v?.value === selectedDataViewId)
+    .filter(isDefined);
+
   return (
     <>
-      <EuiFormRow label="Index" fullWidth>
+      <EuiFormRow
+        label={
+          <FormattedMessage
+            id="xpack.ml.trainedModels.testModelsFlyout.dataViewTitle"
+            defaultMessage="Data View"
```

Comment:
Nit - `Data view` for sentence case here

Reply:
fixed in https://github.com/elastic/kibana/pull/180795/commits/745fdca29d23db2075aa69239486804a83d8203c